### ISSUE TITLE
Linker konnector

### ISSUE THREAD

**Issue by ZeHiro** [cozy-labs/konnectors#299]
This konnector would not import data from a service, but just link a set of files to operations from a given folder, and move them to another folder.
Use case : the konnector for the service is not developped yet, or there will never be. Eg: the company to which I rent my apartment sends me pdf bills, but there will never be a konnector for that (to small company).
Based on the name of the file (eg: YYYYMM_My_Rent_Company.pdf) + an input for my rent amount + operation label, I can use the link_operation function to match the operation and move the files to a target folder., and maybe create a Bill document.
Do you think it could be done @frankrousseau  (not technically, but more if konnectors is the right place for that)

**Comment by frankrousseau** [comment]
Yes definitely. Maybe that konnectors could evolve to "scriptors", a bunch of javascript scripts runned on a regular basis or on demand.

**Comment by ZeHiro** [comment]
Ok, i'll try to do something for that
